src/SFA.DAS.Commitments.Api.Types/SFA.DAS.Commitments.Api.Types.csproj

```diff
@@ -1,139 +1,23 @@
-    <Compile Include="BulkUploadFileRequest.cs" />
-    <Compile Include="Commitment\TransferApprovalRequest.cs" />
-    <Compile Include="Commitment\CommitmentView.cs" />
-    <Compile Include="Commitment\CommitmentRequest.cs" />
-    <Compile Include="Commitment\CommitmentSubmission.cs" />
-    <Compile Include="Commitment\MessageView.cs" />
-    <Compile Include="Commitment\TransferApprovalStatus.cs" />
-    <Compile Include="Commitment\TrainingCourseSummary.cs" />
-    <Compile Include="Commitment\TransferRequest.cs" />
-    <Compile Include="Commitment\TransferRequestSummary.cs" />
-    <Compile Include="Commitment\TransferSender.cs" />
-    <Compile Include="Commitment\TransferType.cs" />
-    <Compile Include="Commitment\Types\MessageCreator.cs" />
-    <Compile Include="ConsistencyStatistics.cs" />
-    <Compile Include="DataLock\DataLockStatus.cs" />
-    <Compile Include="DataLock\DataLocksTriageResolutionSubmission.cs" />
-    <Compile Include="DataLock\DataLockSummary.cs" />
-    <Compile Include="DataLock\DataLockTriageSubmission.cs" />
-    <Compile Include="DataLock\Types\DataLockUpdateType.cs" />
-    <Compile Include="DataLock\Types\DataLockErrorCode.cs" />
-    <Compile Include="DataLock\Types\EventStatus.cs" />
-    <Compile Include="DataLock\Types\Status.cs" />
-    <Compile Include="DataLock\Types\TriageStatus.cs" />
-    <Compile Include="DeleteRequest.cs" />
-    <Compile Include="Commitment\Types\EditStatus.cs" />
-    <Compile Include="Extensions\ApprenticeshipExtensions.cs" />
-    <Compile Include="History\Types\ApprenticeshipChangeType.cs" />
-    <Compile Include="History\ApprenticeshipHistoryItem.cs" />
-    <Compile Include="History\Types\CommitmentChangeType.cs" />
-    <Compile Include="History\CommitmentHistoryItem.cs" />
-    <Compile Include="Commitment\Types\LastAction.cs" />
-    <Compile Include="Commitment\Types\LastUpdateInfo.cs" />
-    <Compile Include="Apprenticeship\Types\PaymentStatus.cs" />
-    <Compile Include="Commitment\Commitment.cs" />
-    <Compile Include="Commitment\CommitmentListItem.cs" />
-    <Compile Include="Commitment\Types\CommitmentStatus.cs" />
-    <Compile Include="OrganisationType.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="ProviderPayment\ProviderPaymentPriorityUpdateItem.cs" />
-    <Compile Include="ProviderPayment\ProviderPaymentPrioritySubmission.cs" />
-    <Compile Include="ProviderPayment\ProviderPaymentPriorityItem.cs" />
-    <Compile Include="Relationship.cs" />
-    <Compile Include="RelationshipRequest.cs" />
-    <Compile Include="Apprenticeship\Types\TrainingType.cs" />
-    <Compile Include="History\Types\UserRole.cs" />
-    <Compile Include="Validation\ApprenticeshipOverlapValidationResult.cs" />
-    <Compile Include="Validation\ApprenticeshipOverlapValidationRequest.cs" />
-    <Compile Include="Validation\OverlappingApprenticeship.cs" />
-    <Compile Include="Validation\Types\ValidationFailReason.cs" />
```

Comment:
As the project now includes all files from the file system, I've checked that no 'orphan' files (in the repo, but not referenced in the old project file) have been accidentally added. All looks good.